_posts/2017-11-24-devopsdays-berlin-2017.md

```diff
@@ -0,0 +1,47 @@
+---
+layout:     post
+title:      devopsdays Berlin 2017
+date:       2018-01-04 15:00:00
```

Comment:
This date doesn't match the date in the filename.  We can fix this just before merging though, so no need to fix it now.